src/components/dashboard/containers/DashboardStats/DashboardStats.js

```diff
@@ -268,9 +268,14 @@ export class DashboardStats extends Component {
 					linkCard
 					onClick={this.goToAccount}
 					subtitle={t('IDENTITY_DAI_BALANCE_AVAILABLE_INFO', {
-						args: [identityBalanceDai, outstandingBalanceDai],
+						args: [identityBalanceDai || 0, outstandingBalanceDai || 0],
```

Comment:
Can you check if the balances are undefined or null and show "N/A" or "Loading...", and `0` if they are '0'

Reply:
OK it has loading ..., that's OK. Just make sure if all balances are 0, to show 0